frameworks/g-cloud-9/questions/services/virtualisationTechnologiesUsed.yml

```diff
@@ -0,0 +1,29 @@
+name: "Virtualisation technologies used"
+question: "What virtualisation technologies are used?"
+depends:
+  -
+    "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+
+
+list_item_name: "virtualisation technologies"
+type: list
+
+validations:
+  -
+    name: answer_required
```

Comment:
This is a follow-up to do you use virtualisation technologies? So we can ask people to 'Describe how different organisations sharing the same infrastructure are kept apart.'